job/models.py

```diff
@@ -3579,10 +3594,38 @@ class ExistingBoilerOutputs(BaseModel, models.Model):
         default = list,
     )
 
+    year_one_thermal_to_tes_series_mmbtu_per_hour = ArrayField(
+        models.FloatField(null=True, blank=True),
+        default = list,
+    )
+
     def clean(self):
         # perform custom validation here.
         pass
 
+class MessagesOutputs(BaseModel, models.Model):
```

Comment:
Done, I have renamed the model for output messages from REoptjl to avoid confusion. Now, we use the existing Messages layout to pass REopt's warnings/errors.

Reply:
I still think that the two approaches should be integrated so we have just one message model, but not essential to do now so I'll create an issue for it.